app/views/posts/new.html.erb

```diff
@@ -1,10 +1,16 @@
-<main>
-  <h1>Add a new post</h1>
-  <%= form_with model: @post, url: user_posts_path, class: "form" do |form| %>
-    <%= form.label :title %> <br>
-    <%= form.text_field :title %> <br>
-    <%= form.label :text %> <br>
-    <%= form.text_area :text , cols: 30, rows: 10 %> <br>
-    <%= form.submit "Submit", class: "btn" %>
-  <% end %>
-</main>
\ No newline at end of file
+<%= form_with(model: [@user, @post], url: user_posts_path(@user), class: 'bg-success d-flex flex-column pt-5 px-5 pb-4'  ) do |f| %>
```

Comment:
- [ ] I could not create posts from the app. When I visit the new post form, it brings errors. Kindly look into that. I created a post from the console to go ahead with the other requirements.

  ![image](https://github.com/Marcraphael12/rails-blog-app/assets/36542986/43b13afd-f79c-4ce0-925f-315d79544170)